src/main/java/com/github/maven_nar/NarCompileMojo.java

```diff
@@ -177,6 +177,22 @@ private void createLibrary(final Project antProject, final Library library)
       }
     }
 
+      if(getOS().equals( OS.WINDOWS ) && getArchitecture().equals("amd64"))
```

Comment:
getAssembler should return null if not available, this test for os seems unecessary. 
Maybe arch also unecessary - why only 64 bit? while 32 bit assembler might generally be inline could it not also be in seperate files?